chain/owasm/res/main/src/lib.rs

```diff
@@ -0,0 +1,19 @@
+extern "C" {
+    pub fn getCurrentRequestID() -> i64;
+    pub fn getRequestedValidatorCount() -> i64;
+
+    pub fn saveReturnData(dataOffset: *const u8, dataLength: i32);
+}
+
+#[no_mangle]
+pub extern "C" fn execute() {
```

Comment:
We already changed to wasm-pack. `pub extern "C"` is needed to use?

Reply:
done